### ISSUE TITLE
Question: K-fold cross validation

### ISSUE THREAD

**Issue by Nylio-prog** [wasserth/TotalSegmentator#183]
Hello,

I was wondering if you tried to train with the k-fold cross validation as nnUNetv2 explained how to do ? Did you not use it because it would take more time for almost no improvements or there are other technical reasons I'm not aware of ? 

Thanks for the clarification.

**Comment by wasserth** [comment]
Using the ensemble of k-fold cross validation would increase the runtime by 5x but only increase accuracy by a very small percentage. Therefore we do not use it.

**Comment by wasserth** [comment]
Since the amount of data is very big and most of the data is used for training, the training is not affected a lot by changing the split. Since the validation set is smaller, it is affected more, but we actually did not use the validation set for anything as we did not tune any hyperparameters. So we do not care. The test set again is more important and therefore bigger and independent from the CV split.

When using more seeds then the question is what to do with the models?? If you ensemble the resulting models you have a lot longer runtime. Therefore we only use one seed for faster runtime.
